### DISCUSSION TITLE
Issue while import Airtable to NocoDB [Jun 01]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#2298] [Discord Threads]
> 🙋‍♂️ **LuizVaz**

**Comment by o1lab** [comment]
> 💁 **o1lab**  
#### could you plz project info (right top menu)  
#  
> 💁 **o1lab**  
#### also please confirm you are entering   
#### * API key and    
#### * shared vase id correctly  
#  
> 💁 **o1lab**  
#### https://docs.nocodb.com/setup-and-usages/import-airtable-to-sql-database-within-a-minute-for-free  
#  
> 🙋‍♂️ **LuizVaz**  
#### Yes the api key is correct and shared base id too.   
#### ```   
#### Node: **v12.22.12**   
#### Arch: **x64**   
#### Platform: **linux**   
#### Docker: **true**   
#### Database: **pg**   
#### ProjectOnRootDB: **true**   
#### RootDB: **pg**   
#### PackageVersion: **0.91.0**   
#### ```  
#  
> 💁 **o1lab**  
#### have DMed you  
#  
> 💁 **o1lab**  
#### Please try the latest release v0.91.1   
####    
#### We've made improvements  
#  
> 🙋‍♂️ **LuizVaz**  
#### Thanks, guys.   
#### You are awesome!   
#### Problem solved with the latest release.  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/979108957010423888)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
